src/UraniumUI.Material/Controls/TextField.cs

```diff
@@ -148,4 +150,28 @@ public override void ResetValidation()
         EntryView.Text = string.Empty;
         base.ResetValidation();
     }
+
+    #region DisallowClearButtonFocus Logic
+
+    protected void ValidateClearButtonFocus()
```

Comment:
We can implement it in platform-specific via accessing `iconClear.Handler.PlatformView`, it may work more fluent. 

Probably it's `Panel` on windows and is has IsTabStop property on it. I don'T know what is on MacCatalyst but I can check

Reply:
Don't have a Mac to test that development

Reply:
Done and tested on windows/android.

Couldnt test Mac/IOS but it should be correct.

Reply:
Perfectly done! Thanks, I'll check on the mac before merging it 🎉